packages/backend/src/api/controllers/report/generateMain.ts

```diff
@@ -11,23 +10,17 @@ export function generateMain(
   sixHourlyReports: AggregateReportRecord[],
   dailyReports: AggregateReportRecord[],
   latestReports: ReportRecord[],
-  projects: ReportProject[],
+  projectIds: ProjectId[],
 ): ApiMain {
+  const reports = [hourlyReports, sixHourlyReports, dailyReports] as const
```

Comment:
Maybe instead of destructuring this for every call of `getProjectCharts` we'd make `getProjectCharts` accept an object `reports = { hourly, sixHourly, daily }`?